packages/react-native/ios/vendor/bugsnag-cocoa/Bugsnag.podspec.json

```diff
@@ -9,7 +9,7 @@
   },
   "source": {
     "git": "https://github.com/bugsnag/bugsnag-cocoa.git",
-    "tag": "v6.1.0"
+    "tag": "v6.1.1"
```

Comment:
6.1.2 is imminent - did you want to handle that separately?

Reply:
I landed the changes from the `react-native-plugin` integration branch, are the 6.1.2 changes not on that?

Reply:
Sorry, got the wrong end of the stick as to how the update script works.